layer/layer.go

```diff
@@ -173,7 +173,7 @@ type Store interface {
 	Map() map[ChainID]Layer
 	Release(Layer) ([]Metadata, error)
 
-	CreateRWLayer(id string, parent ChainID, mountLabel string, initFunc MountInit, storageOpt map[string]string) (RWLayer, error)
+	CreateRWLayer(id string, parent ChainID, mountLabel string, initFunc MountInit, storageOpt map[string]string, sharedRootFs bool) (RWLayer, error)
```

Comment:
This function interface should not be updated. Can you add an interface similar to `DescribableStore`, maybe called `ShareableStore` with a `CreateSharedRWLayer` and assert to use it.  I don't have a problem with the underlying `layerStore` implementing this inteface.